api/app/controllers/spree/api/v1/checkouts_controller.rb

```diff
@@ -3,6 +3,7 @@ module Api
     module V1
       class CheckoutsController < Spree::Api::BaseController
         before_action :associate_user, only: :update
+        before_action :load_order_lock_true, only: [:next, :advance, :update]
```

Comment:
@jaspreet21anand - Also, load_order_with_lock seems a better name for this action, what say?